src/mongocxx/write_concern.hpp

```diff
@@ -54,6 +54,15 @@ class bulk_write;
 class MONGOCXX_API write_concern {
 
    public:
+    ///
+    /// A class to represent the special case values for write_concern::nodes.
+    /// @see http://docs.mongodb.org/manual/reference/write-concern/#w-option
+    ///
+    enum w : std::int32_t {
+        UNACKNOWLEDGED = 0,
```

Comment:
our naming scheme for enum values is to prefix them with `k` and use camelCase.

e.g. kUnacknowledged, kDefault, kMajority...